website/docs/d/data_lake_store.html.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_data_lake_store"
+sidebar_current: "docs-azurerm-datasource-data-lake-store"
+description: |-
+  Get information about a Data Lake Store
+
+---
+
+# Data Source: azurerm_data_lake_store
+
+Use this data source to obtain information about a Data Lake Store.
+
+## Example Usage
+
+```hcl
+data "azurerm_data_lake_store" "test" {
+  name                = "testdatalake"
+  resource_group_name = "testdatalake"
+}
+
+output "data_lake_store_id" {
+  value = "${data.azurerm_data_lake_store.test.id}"
+}
+```
+
+## Argument Reference
+
+* `name` - (Required) The name of the DNS Zone.
```

Comment:
I think this should be 'The name of the Data Lake Store'

Reply:
yes :), fixed